build.sbt

```diff
@@ -69,7 +69,7 @@ val common = library("common")
       pekkoSlf4j,
       pekkoSerializationJackson,
       pekkoActorTyped,
-    ),
+    ) ++ jackson,
```

Comment:
This adds the forced jackson dependency to the "common" sub-project, which is used by (depended on by) all the application sub-projects below. However, it isn't used by the root sub-project! The root project also uses Play, so it will still be pulling in a vulnerable version of jackson without this override included there as well.

Reply:
Thanks @adamnfish! This worked 🪄